packages/node_modules/nav-frontend-skjema/src/skjema-gruppe.tsx

```diff
@@ -55,18 +55,31 @@ export interface SkjemaGruppeProps {
  */
 class SkjemaGruppe extends React.Component<SkjemaGruppeProps> {
     render() {
+        let feilmeldingId = this.props.feilmeldingId;
         const {
             children,
             className,
             feil,
             legend,
             description,
             tag = 'fieldset',
-            feilmeldingId,
             utenFeilPropagering,
-            ...other
         } = this.props;
+
+        const domprops = omit(
```

Comment:
Trenger vi å gjøre destrukturering og omit for de samme propsene?
Ville ikke noe alà dette fungert;
```
const {
  children,
  className,
  feil,
  legend,
  description,
  tag = 'fieldset',
  utenFeilPropagering,
  ...other
} = this.props;

const feilmeldingId = this.props.feilmeldingId || guid();
const domprops = omit(other, 'feilmeldingId');
```